app/models/in_person_enrollment.rb

```diff
@@ -151,6 +151,10 @@ def usps_unique_id
     user.uuid.delete('-').slice(0, 18)
   end
 
+  def enhanced_ipp?
+    IdentityConfig.store.usps_eipp_sponsor_id == sponsor_id
```

Comment:
I ran the test instructions, and happened to not have `usps_eipp_sponsor_id` or `sponsor_id` set in my application.yml, and got a `NoMethodError: undefined method '[]' for nil` when running `job.send(:process_enrollment_response, e, res)`. Then when inspecting the enrollment I observed it was `pending`. When I went and set those variables and tried the test again, I observed the expected behavior of the enrollment being `failed`.

So I'm wondering:

Does an additional check need to be added that these values are not empty here?

Does it make sense to add default dummy values to these variables in application.yml.default?

Reply:
@eileen-nava  What are you thoughts? I have mixed feelings on both and would like to hear what others think.

Reply:
Added a default in development. I think this is necessary because I think the likely hood of this being set in application.yml is slim by developers outside of Team Joy.  

Also, please confirm my understanding- what is added in via [app-s3-secret](https://handbook.login.gov/articles/appdev-secrets-configuration.html) overrides this value- so because these values are set in all dev environments this will likly only be used locally if values are not set in application.yml.

Reply:
I agree that developers who aren't on Team Joy probably won't have `usps_eipp_sponsor_id` set in their personal `application.yml` files.

Re: the second paragraph, your understanding matches my understanding.  I think the second point in `Configuration in Rails Apps` in the docs you linked is also relevant.
> 2. Configure a default value in config/application.yml.default at the top level of the file. If there is no value specified in S3 for this config, this default value will be used in production.

Reply:
I like the addition of the default values! 👍